ddcommon-ffi/src/container_id.rs

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use ddcommon::container_id::get_container_id;
+use std::ffi::CString;
+
+use std::os::raw::{c_char, c_int};
+
+#[repr(C)]
+pub struct FfiOptionString {
```

Comment:
@morrisonlevi I'm stlightly confused by the different FFI objects.
If you can help with the FFI API here. An optional CharSlice ?